gambit/1.0.0/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Software installation, no database files
+FROM ubuntu:xenial as base
+
```

Comment:
I'd hate to have you rewrite the dockerfile, but as I was looking through the installation, I wonder if using the micromamba base would be simpler.

Here's some examples: 
- https://github.com/StaPH-B/docker-builds/blob/master/poppunk/2.5.0/Dockerfile
- https://github.com/StaPH-B/docker-builds/blob/master/ngmaster/0.5.8/Dockerfile

(not necessary, just a suggestion)

Reply:
Done